### PR TITLE
Add migration notes and update v5 scripting

### PR DESCRIPTION
**Pull request by jasonatball**

### CODE REVIEW

File: _docs_v5/4_guides/scripting.md [hunk 1]
```diff
@@ -246,23 +246,130 @@ Next is a tool bar that displays the currently executing script and three button
...
+| close_local_screens        | Telemetry Viewer | Unimplemented |
+| cmd_tlm_clear_counters     | Command and Telemetry Server | Unimplemented |
+| cmd_tlm_reload             | Command and Telemetry Server | Unimplemented |
+| display                    | Telemetry Viewer | Unimplemented |
+| get_all_packet_logger_info | Command and Telemetry Server | Deprecated |
+| get_background_tasks       | Command and Telemetry Server | Deprecated |
+| get_cmd_list               | Command and Telemetry Server | Deprecated, use get_all_commands |
+| get_cmd_log_filename       | Command and Telemetry Server | Deprecated |
+| get_cmd_param_list         | Command and Telemetry Server | Deprecated use get_command |
+| get_all_interface_info     | Command and Telemetry Server | Deprecated, use get_inter |
```
Comment (ryanatball) [inline _docs_v5/4_guides/scripting.md:273]: typo

File: _docs_v5/4_guides/scripting.md [hunk 2]
```diff
@@ -246,23 +246,130 @@ Next is a tool bar that displays the currently executing script and three button
...
+| local_screen | Script Runner | Unimplemented |
+| map_target_to_interface | Command and Telemetry Server | Unimplemented |
+| override_tlm_raw  | Command and Telemetry Server | Deprecated, use override_tlm |
+| open_file_dialog | Script Runner | Unimplemented |
+| open_files_dialog | Script Runner | Unimplemented |
+| open_directory_dialog | Script Runner | Unimplemented |
+| prompt_combo_box | Script Runner | Unimplemented |
+| prompt_message_box | Script Runner | Unimplemented |
+| prompt_to_continue | Script Runner | Unimplemented |
+| prompt_vertical_message_box | Script Runner | Unimplemented |
```
Comment (ryanatball) [inline _docs_v5/4_guides/scripting.md:306]: I think prompt_* are all internal methods, not part of the user facing api.

File: _docs_v5/4_guides/scripting.md [hunk 3]
```diff
@@ -246,23 +246,130 @@ Next is a tool bar that displays the currently executing script and three button
...
+| router_state | Command and Telemetry Server | Deprecated, use get_router |
+| run_mode | Script Runner | Unimplemented |
+| save_file_dialog | Script Runner | Unimplemented |
+| script_disconnect | Script Runner | Unimplemented |
+| set_disconnected_targets | Script Runner | Unimplemented |
+| set_replay_mode | Replay | Deprecated |
+| set_stdout_max_lines | Script Runner | Deprecated |
+| set_tlm_raw  | Script Runner | Deprecated, use set_tlm |
+| show_backtrace | Script Runner | Unimplemented |
+| shutdown_cmd_tlm | Command and Telemetry Server | Deprecated |
```
Comment (ryanatball) [inline _docs_v5/4_guides/scripting.md:327]: I think this still exists in script?

File: _docs_v5/4_guides/scripting.md [hunk 4]
```diff
@@ -246,23 +246,130 @@ Next is a tool bar that displays the currently executing script and three button
...
+| replay_step_back | Replay | Deprecated |
+| replay_step_forward | Replay | Deprecated |
+| replay_stop | Replay | Deprecated |
+| router_state | Command and Telemetry Server | Deprecated, use get_router |
+| run_mode | Script Runner | Unimplemented |
+| save_file_dialog | Script Runner | Unimplemented |
+| script_disconnect | Script Runner | Unimplemented |
+| set_disconnected_targets | Script Runner | Unimplemented |
+| set_replay_mode | Replay | Deprecated |
+| set_stdout_max_lines | Script Runner | Deprecated |
```
Comment (ryanatball) [inline _docs_v5/4_guides/scripting.md:324]: I don't remember this, but might still want?  I think this clipped stdout at some size to prevent lockups

**Review by ryanatball — CHANGES_REQUESTED** [review]

File: _docs_v5/4_guides/scripting.md (hunk 2 above)
Comment (jasonatball) [reply _docs_v5/4_guides/scripting.md:306]: Will remove

File: _docs_v5/4_guides/scripting.md (hunk 3 above)
Comment (jasonatball) [reply _docs_v5/4_guides/scripting.md:327]: I couldn't find it and what would that even mean in COSMOS 5? We can't shut down the cmd/tlm server for all tools via script.

File: _docs_v5/4_guides/scripting.md (hunk 4 above)
Comment (jasonatball) [reply _docs_v5/4_guides/scripting.md:324]: I'll mark as unimplemented then

**Pull request merged**